tendrl/gluster_integration/sds_sync/vol_utilization.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/python
+
+import json
+import sys
+import argparse
+
+from tendrl.gluster_integration import gfapi
+
+
+BYTES_IN_KB = 1024
+
+
+def computeVolumeStats(data):
+    total = data.f_blocks * data.f_bsize
+    free = data.f_bfree * data.f_bsize
+    used = total - free
+    return {'sizeTotal': float(total),
+            'sizeFree': float(free),
+            'sizeUsed': float(used)}
+
+
+def showVolumeUtilization(vname):
+    try:
+        data = gfapi.getVolumeStatvfs(vname)
+    except gfapi.GlusterLibgfapiException:
+        sys.stdout.write("CRITICAL: Failed to get the "
```

Comment:
if we write to stdout, will it report as err ? should we write to stderr ?
Also, it would be better to write the exception info as well because we will miss out why the script failed.

Reply:
I had changed it. I think I forgot to push. Will do that.